src/main/kotlin/net/atos/client/zgw/ztc/model/RoltypeListParameters.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.zgw.ztc.model
+
+import jakarta.ws.rs.QueryParam
+import net.atos.client.zgw.ztc.model.generated.RolType.OmschrijvingGeneriekEnum
+import java.net.URI
+
+class RoltypeListParameters(
+    private val zaaktype: URI,
```

Comment:
I think you can use (similar as done on other fields) the annotation `@field:QueryParam("zaaktype")` directly in the constructor.

Reply:
No, see above. That did not work..